subprojects/docs/src/docs/userguide/running-builds/introduction/build_scans.adoc

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[build_scans]]
+= Build Scans
+
+A build scan is a *representation of data captured* as you run your build.
+
+image::gradle-basic-8.png[]
```

Comment:
This image doesn't seem to relate to build scans. It still highlights the build cache block